docs/cimplInstall.md

```diff
@@ -34,6 +34,7 @@ setx https_proxy https://your.proxy.org
 setx proxy http://your.proxy.org  
 setx JAVA_OPTS "-Dhttp.proxyHost=your.proxy.org -Dhttp.proxyPort=80 -Dhttps.proxyHost=your.proxy.org -Dhttps.proxyPort=80 -DsocksProxyHost=your.proxy.org -DsocksProxyPort=80"
```

Comment:
I don't know that we need the JAVA_OPTS environment setting anymore.  cc: @cmoesel to weigh in.

Reply:
It's still needed if you're in an environment w/ an explicit proxy.  At MITRE the macs have been transitioned to transparent proxy, so we don't need it anymore -- but windows are still using the explicit proxy.

All that said, last time I checked, IG Publisher wasn't honoring the Java proxy settings anyway. :-/

Reply:
Do I leave it in the install documentation or not?  I ran this command today without JAVA_OPTS in a windows environment and it worked although the output I get isn't as expected.